src/test/java/org/folio/test/support/matcher/AccountMatchers.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.test.support.matcher;
+
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.is;
+
+import org.hamcrest.Matcher;
+
+import io.restassured.response.Response;
+
+public final class AccountMatchers {
+
+  private AccountMatchers() {}
+
+  public static Matcher<Response> isAccountPaidFully() {
```

Comment:
I imagine that `account` is implicit in the context of the use of this matcher, and so might it read better without that in the matchers name e.g. `account is paid fully` rather than `account is account paid fully`

What do you think?

```suggestion
  public static Matcher<Response> isPaidFully() {
```

Reply:
renamed.